### PR TITLE
Pin package versions centrally using Directory.Build.targets

### PR DESCRIPTION
**Pull request by filipw**
To avoid package versions diverging between projects.

### CODE REVIEW

**Comment by filipw** [comment]
ping @khellang @jrusbatch

File: Directory.Build.targets [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Update="Autofac" Version="4.6.2" />
+    <PackageReference Update="Autofac.Mef" Version="4.0.0" />
+    <PackageReference Update="McMaster.Extensions.CommandLineUtils" Version="2.2.5" />
+	<PackageReference Update="Microsoft.CodeAnalysis.Scripting" Version="2.9.0" />
```
Comment (khellang) [inline Directory.Build.targets:6]: Nit: indentation

**Review by khellang — APPROVED** [review]
LGTM! No idea what the `Paket.Restore.targets` changes are though...

**Review by jrusbatch — APPROVED** [review]
👍

**Comment by filipw** [comment]
> No idea what the `Paket.Restore.targets` changes are though...

yeah.. new version of Paket did them automatically and apparently that file should be in source control ¯\_(ツ)_/¯ https://github.com/fsprojects/Paket/issues/2840#issuecomment-342737075

**Pull request merged**